### PR TITLE
fix for IE browser issue recognition of object.assign() + jQuery vulnerability fix + minor updates

### PR DESCRIPTION
**Pull request by HIGGO36**
#### What's this PR do?

There are a few different updates within this:

- fix for an existing IE browser compatibility when using the object.assign() method by installing the "@babel/plugin-transform-object-assign": "7.2.0", and updating babel's configuration settings within the "rollup-config-helper.js" file by adding the new plugin option below it's preset options. 

- Updated jQuery core to 3.4.1 version to fix the vulnerability issues dealing with prototype pollution

- Fixed some of the syntax issues within the index.html file

- added css rule providing a margin-bottom to all forms, to help give space between them

#### Where should the reviewer start?

Can compare the current live kitchen sink with the updated version within the IE browser, and look at certain components that are having issues, ex: DatePicker, Select2, Selectize, etc..

#### How should this be manually tested?

using IE browser to view functionality of components with the kitchen sink and running audit to see if the jQuery vulnerability is handled with version 3.4.1.

#### Any background context you want to provide?

We wanted to make sure and provide all fixes for IE browser after updating to Babel 7, and continue to improve upon the overall theme.

#### What are the relevant tickets?

n/a

#### Screenshots (if appropriate)

n/a

#### Questions:
- Is there a blog post?
n/a

- Does the knowledge base need an update?

maybe

### CODE REVIEW

File: src/styles/custom/_forms.scss [hunk 1]
```diff
@@ -64,6 +64,11 @@ span.file-title {
 }
 // end of input file type styling
 
+// General FORM styling needs
+form {
```
Comment (davezen1) [inline src/styles/custom/_forms.scss]: why is this needed? If it is a styling update we should have a separate ticket for it.

File: kitchensink/kitchensink.css [hunk 2]
```diff
@@ -89,35 +89,3 @@
     visibility: visible;
   }
 }
-
-
```
Comment (davezen1) [inline kitchensink/kitchensink.css]: This does not seem related to the IE fix

File: src/styles/custom/_forms.scss (hunk 1 above)
Comment (HIGGO36) [reply src/styles/custom/_forms.scss]: agreed, not necessary at this point, will remove this from pr.

Comment (HIGGO36) [reply src/styles/custom/_forms.scss]: ok, was having issue removing this from this pr's files directly on github, so instead updated locally and pushed.

File: kitchensink/kitchensink.css (hunk 2 above)
Comment (HIGGO36) [reply kitchensink/kitchensink.css]: left this as is, since just removed commented out code.

**Comment by HIGGO36** [comment]
i also updated the index file back to original, with only the update for the reference to new jquery 3.4.1 min file.

**Comment by HIGGO36** [comment]
> See the comments that mention styling changes. We can create a ticket for any style changes that need to be made.

cool, will definitely follow this process if we decide to make any more styling changes in future.
